tests/test_butler.py

```diff
@@ -1590,7 +1590,6 @@ def setUp(self):
 
 
 @unittest.skipIf(not boto3, "Warning: boto3 AWS SDK not found!")
-@mock_s3
```

Comment:
This looks like it follows the documentation, but it's a little unclear how inherited tests are handled.

Since this code works, I guess it's OK.

Reply:
Yes, the exact same mocking works fine in the `resources` package but that doesn't inherit tests from another class.